integration-test/src/test/java/org/apache/iotdb/db/it/schema/IoTDBCreateAlignedTimeseriesIT.java

```diff
@@ -94,7 +93,7 @@ public void testCreateAlignedTimeseries() throws Exception {
   public void testCreateAlignedTimeseriesWithDeletion() throws Exception {
     String[] timeSeriesArray =
         new String[] {
-          "root.sg1.d1.vector1.s1,DOUBLE,PLAIN,SNAPPY", "root.sg1.d1.vector1.s2,INT64,RLE,SNAPPY"
+          "root.sg1.d1.vector1.s1,DOUBLE,PLAIN,SNAPPY", "root.sg1.d1.vector1.s2,INT64,RLE,LZ4"
```

Comment:
Why `root.sg1.d1.vector1.s1` still be  `SNAPPY`?

Reply:
Compressor=SNAPPY is used when creating root.sg1.d1.vector1.s1